packages/e2e-tests/src/specs/editor/taxonomy.js

```diff
@@ -76,95 +77,130 @@ async function addCategory(name, parent) {
 
-  it.todo('should be able to add new tags and existing tags');
+    it.todo('should be able to add new tags and existing tags');
 
-  describe('Contributor', () => {
-    withUser('contributor', 'password');
+    describe('Contributor', () => {
+      withUser('contributor', 'password');
 
-    it('should be able to manage categories but not add new ones', async () => {
-      await createNewStory();
-      await insertStoryTitle('Taxonomies - Categories - Contributor');
+      it('should be able to manage categories but not add new ones', async () => {
+        await createNewStory();
+        await insertStoryTitle('Taxonomies - Categories - Contributor');
 
-      await goToAndExpandTaxonomyPanel();
+        await goToAndExpandTaxonomyPanel();
 
-      await expect(page).not.toMatchElement('button', {
-        text: 'Add New Category',
-      });
+        await expect(page).not.toMatchElement('button', {
+          text: 'Add New Category',
+        });
 
-      await expect(page).toMatchElement('input[name="hierarchical_term_rock"]');
+        await expect(page).toMatchElement(
+          'input[name="hierarchical_term_rock"]'
+        );
 
-      await expect(page).toClick('label', { text: 'rock' });
+        await expect(page).toClick('label', { text: 'rock' });
 
-      await expect(page).toClick('button[aria-label="Save draft"]');
-      await page.waitForSelector(
-        'button[aria-label="Preview"]:not([disabled])'
-      );
+        await expect(page).toClick('button[aria-label="Save draft"]');
+        await page.waitForSelector(
+          'button[aria-label="Preview"]:not([disabled])'
+        );
 
-      // Refresh page to verify that the assignments persisted.
-      await page.reload();
-      await expect(page).toMatchElement('input[placeholder="Add title"]');
-      await expect(page).toMatchElement('a.ab-item', { text: 'View Story' });
+        // Refresh page to verify that the assignments persisted.
+        await page.reload();
+        await expect(page).toMatchElement('input[placeholder="Add title"]');
+        await expect(page).toMatchElement('a.ab-item', { text: 'View Story' });
 
-      await goToAndExpandTaxonomyPanel();
+        await goToAndExpandTaxonomyPanel();
 
-      await expect(page).toMatchElement(
-        'input[name="hierarchical_term_rock"][checked]'
-      );
+        await expect(page).toMatchElement(
+          'input[name="hierarchical_term_rock"][checked]'
+        );
 
-      await percySnapshot(page, 'Taxonomies - Categories - Contributor');
-    });
+        await percySnapshot(page, 'Taxonomies - Categories - Contributor');
+      });
 
-    it.todo(
-      'should be able to add tags that already exist but not create new tags'
-    );
+      it.todo(
+        'should be able to add tags that already exist but not create new tags'
+      );
+    });
+  });
+  describe('Custom Taxonomy', () => {
+    withPlugin('web-stories-test-plugin-taxonomies');
+    describe('Administrator', () => {
```

Comment:
Nit: Can't hurt to give some breathing room

```suggestion

  describe('Custom Taxonomy', () => {
    withPlugin('web-stories-test-plugin-taxonomies');
    
    describe('Administrator', () => {
```